src/config/texts.js

```diff
@@ -956,7 +956,10 @@ export const texts = {
           title: 'Bildquelle auswählen'
         },
         invalidMail: 'Die eingegebene E-Mail-Adresse ist nicht gültig.',
+        limitOfArea: (city) =>
+          `Leider kann die Sag's uns Einfach nur Standorte der ${city} nutzen.`,
```

Comment:
this text shouldn't be hardcoded to `Sag's uns Einfach`

Reply:
no, needs to be discussed with others. if it is always the app name, we can use the config from app.json?

Reply:
Yes, we can definitely use it. That's what I thought. But I'll put it in my notes to ask at the next meeting.

Reply:
We talked about it and decided to leave it as it is. Because this text can actually be set on the mainserver.

Reply:
Ok, i was not aware about the possibility to configure this text.